src/corehost/cli/deps_resolver.cpp

```diff
@@ -486,6 +486,16 @@ void deps_resolver_t::resolve_probe_dirs(
     {
         if (probe_entry_in_configs(entry, &candidate))
         {
+            // For standalone apps, on win7, coreclr needs ApiSets which has to be in the DLL search path.
+            if (!m_portable && pal::need_api_sets()
```

Comment:
> What if newer clr would need dll's from another package?

APISets are the only CoreCLR dependency that are not present in the system (i.e. pre-req since they cannot be).